lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -120,7 +122,7 @@
     platform.registerFunction(FunctionsEnum.ArrayEntries, function () {
         "use strict";
         if (this === null || this === undefined) {
-            throw new TypeError("Array.prototype.entries: 'this' is null or undefined");
+            __chakraLibrary.raiseThis_NullOrUndefined("Array.prototype.entries");
         }
```

Comment:
I could not use platform.raiseThis_NullOrUndefined the same was as Intl uses, Intl maybe doing something special to hold reference to the platform object defined in the parent initialization closure code. Adding error functions on the always available chakraLibrary object.

Reply:
@curtisman does this change look good to unblock you for now ?

Reply:
yep

---
In reply to: [179288292](https://github.com/Microsoft/ChakraCore/pull/4934#discussion_r179288292) [](ancestors = 179288292)